SwitchSpeak-iOS/SwitchSpeak-iOS/Tree.swift

```diff
@@ -10,57 +10,53 @@ import Foundation
 
 import UIKit
 
-
 class Tree {
 	var rootNode: Node?
 	var treeType: ScanType?
 	var size: Int?			//	contains the number of button nodes present in the tree
 	
-	init(root: Node){
+	init(root: Node) {
 		self.rootNode = root
 		self.treeType = nil
 		self.size = 0
 	}
 	
-	init(){
+	init() {
 		let root = Node()
 		self.rootNode = root
 		self.treeType = nil
 		self.size = 0
 	}
 	
 	/*
-	Intput is the grid dimensions of the view, the width and height of the screen, the height of the top bar, and the max button zie dimensions
-	In this function we resize each button in the tree so as to fit it entirely in the view
+		Intput is the grid dimensions of the view, the width and height of the screen, the height of the top bar, and the max button dimensions
+		In this function we resize each button in the tree so as to fit it entirely in the view
 	*/
-	func setFrameSizeForTree (row: Int, col: Int, screenWidth: Int, screenHeight: Int, topBarHeight: Int, maxButtonHeight: Int, maxButtonWidth: Int)
-	{
+	func setFrameSizeForTree (row: Int, col: Int, screenWidth: Int, screenHeight: Int, topBarHeight: Int, maxButtonHeight: Int, maxButtonWidth: Int) {
 		//	first we compute the size of buttons to be placed in view
 		let buttonSize = computeButtonSize(row: row, col: col, width: screenWidth, height: screenHeight-topBarHeight, maxButtonHeight: maxButtonHeight, maxButtonWidth: maxButtonWidth)
 		
-		if (self.size == row*col && self.treeType == .LINEAR)	//	check if the tree is appropriate dimensions and is linear scan type (a depth 2 tree)
-		{
-			for i in 0...row-1{
-				for j in 0...col-1{
-					(self.rootNode?.childNodes[col*i+j] as? ButtonNode)?.button.frame = CGRect(x: (buttonSize.buttonWidth + buttonSize.colGap) * j + buttonSize.colGap, y:  topBarHeight + (buttonSize.buttonHeight + buttonSize.rowGap) * i + buttonSize.rowGap, width: buttonSize.buttonWidth, height: buttonSize.buttonHeight)
+		//	check if the tree is having the appropriate dimensions and is linear scan type (a depth 2 tree)
+		if (self.size == row*col && self.treeType == .LINEAR) {
```

Comment:
Since you're also fixing that little typo, please just stick a space around the operator here, and between the minus signs below. Just to stay consistent. Sorry to be so nit-picky.